packages/dashboard/src/API.tsx

```diff
@@ -12,7 +12,7 @@ export interface PluginServices {
     Wallet: typeof import('../../maskbook/dist/plugins/Wallet/messages').WalletRPC
```

Comment:
```suggestion
    Wallet: typeof import('@masknet/plugin-wallet').WalletRPC
```

Reply:
why not 
```ts
import type { WalletRPC, WalletMessages } from '@masknet/plugin-wallet'
```

Reply:
No, RPC didn't move to the `@masknet/plugin-wallet` package.